server/controllers/konnectors.coffee

```diff
@@ -77,6 +50,24 @@ module.exports =
 
             res.status(204).send konnector
 
+    update: (req, res, next) ->
+        # Don't update connector if an import is already running.
+        if req.konnector.isImporting
+            res.send 400, message: 'konnector is importing'
```

Comment:
HTTP status 400 means _Bad Request_. But here the request is correct. Maybe a 409, _Conflict_ sould be more accurate ? What do you think ?